### PR TITLE
Add 1340. Jump Game V

### LINKED ISSUE #542: 1340 - Jump Game V
## Description of the Problem
You may copy that from LeetCode
Given an array of integers `arr` and an integer `d`. In one step you can jump from index i to index:

- `i + x` where: `i + x < arr.length` and `0 < x <= d`

- `i - x` where: `i - x >= 0` and `0 < x <= d`

In addition, you can only jump from index `i` to index `j` if `arr[i] > arr[j]` and `arr[i] > arr[k]` for all indices k between `i` and `j` (More formally `min(i, j) < k < max(i, j)`).

You can choose any index of the array and start jumping. Return the maximum number of indices you can visit.

Notice that you can not jump outside of the array at any time.

## Code
```
class Solution(object):
    def maxJumps(self, arr, d):
        """
        :type arr: List[int]
        :type d: int
        :rtype: int
        """
```

## Link To The LeetCode Problem
[LeetCode](https://leetcode.com/problems/jump-game-v/)

### PR DESCRIPTION
**Pull request by livace**
**IMPORTANT: Please do not create a Pull Request without an issue.**

### All Submissions:

* [x] I have read the CONTRIBUTING document.
* [x] My code is written in **Python3** and is ending with _.py_
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] I have checked that my submission does pass the test on LeetCode.com 
* [x] Does your filename follow the naming Conventions?
* [x] Have you linked your PR to an Issue?

**Closing issues**

closes #542

### CODE REVIEW

File: LeetCode/1340_Jump_Game_V.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        """
+        :type arr: List[int]
+        :type d: int
+        :rtype: int
+        """
+
+        dp = [1] * len(arr)
+
+        for i, _ in sorted(enumerate(arr), key=lambda x: x[1]):
+            max_h = None
```
Comment (vJechsmayr) [inline LeetCode/1340_Jump_Game_V.py]: Code does not pass Test on LeetCode

image

**Review by vJechsmayr — CHANGES_REQUESTED** [review]
Please check code again - Runtime Error

**Comment by livace** [comment]
Oh, my fault. Now everything should be fine

**Review by vJechsmayr — APPROVED** [review]
Thanks for contributing - Merged!

Runtime: 768 ms

Happy Hacktoberfest!

**Pull request merged**